src/app/interfaces/chats.d.ts

```diff
@@ -0,0 +1,6 @@
+export default interface MessageInterface {
+  userId?: string;
```

Comment:
Why is this optional? I don't understand

Reply:
It was before when the chat controller was giving the  messages an useId, but we changed that logic. Just forgot to change it here